### DISCUSSION TITLE
error in legacey migration tool

### DISCUSSION THREAD

**Discussion by josdeklosBE** [iotaledger/legacy-migration-tool#135] [Help & Support]
after password i got this error:

App Version: 1.0.0

platform: win32
platformVersion: 10.0.22631
platformArchitecture: x64
cpuCount: 16
totalMem: 16277.0 MB
freeMem: 6612.8 MB

Error Type: [Render Context] Unhandled Rejection

Error: Cannot read property 'split' of undefined

**Comment by begonaalvarezd** [comment]
Hi, how did it happen? I would appreciate a bit more information to be able to detect the bug, thanks 🙏🏼
